_episodes/020-mun.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Mun" 
+date: 2020-06-26T17:00:00+00:00
+file: https://audio.rustacean-station.org/file/rustacean-station/mun #are we naming the files ourselves or is the uploader doing it?
```

Comment:
If you want a particular file name, then let me know, otherwise I'll just choose one :p